tests/test_als_explicit.py

```diff
@@ -82,6 +82,80 @@ def test_als_predict_basic(m):
     assert preds.loc[3] >= -0.1
     assert preds.loc[3] <= 5.1
 
+@methods
```

Comment:
I don't think we need to parameterize each of these over `@methods`. We can just use the default solver to train the initial model.